conjurapi/resource_test.go

```diff
@@ -96,20 +98,24 @@ func TestClient_CheckPermission(t *testing.T) {
 }
 
 func TestClient_Resources(t *testing.T) {
-	listResources := func(conjur *Client, filter *ResourceFilter) func() {
+	listResources := func(conjur *Client, filter *ResourceFilter, int expected) func() {
```

Comment:
`expected int` ?

Reply:
Can't test since the image `conjur-appliance-cuke-master` is not available hence `docker-compose` fails

Reply:
@alsastre sorry i should have been clearer - `expected int` as opposed to `int expected`, as in `func (arg_name type, ...)`

Reply:
`./test.sh oss` and `./dev.sh oss` should not require the use of cuke-master unless something is off

Reply:
```
(feature/new_filters_resources) ~/.../alsastre/conjur-api-go $ ./test.sh 
+ trap finish EXIT
+ echo 'Running tests'
Running tests
+ rm -rf output
+ mkdir -p output
+ source ./_setup.sh
+++ sed 's/^[_-]*\(.*\)/\1/'
+++ basename /home/alejandro/Dropbox/Projects/Go/src/github.com/alsastre/conjur-api-go
++ export COMPOSE_PROJECT_NAME=conjur-api-go
++ COMPOSE_PROJECT_NAME=conjur-api-go
++ export TEST_VERSION=all
++ TEST_VERSION=all
++ oss_only
++ '[' all == oss ']'
++ docker-compose pull conjur cuke-master
Pulling conjur      ... done
Pulling cuke-master ... error

ERROR: for cuke-master  Get https://registry2.itci.conjur.net/v2/conjur-appliance-cuke-master/manifests/4.9-stable: unauthorized: authentication required
ERROR: Get https://registry2.itci.conjur.net/v2/conjur-appliance-cuke-master/manifests/4.9-stable: unauthorized: authentication required
+ finish
+ docker-compose down -v
Removing network conjur-api-go_default
WARNING: Network conjur-api-go_default not found.
```

Reply:
> @alsastre sorry i should have been clearer - `expected int` as opposed to `int expected`, as in `func (arg_name type, ...)`

:+1: Changed. Problem of changing between python and go

Reply:
> `./test.sh oss` and `./dev.sh oss` should not require the use of cuke-master unless something is off

Looking at dev and test,, both load `_setup.sh` the variable `TEST_VERSION=` was not corrreclty set. After forcing it to be `oss` I am able to test. 
AFTER TESTING:
Have fixed some errors and now it passes all the test

Reply:
Changing the following in test.sh allows the test to work: 
```
-source ./_setup.sh
+source ./_setup.sh oss
 ```
I suppose `oss` could be substituted by `$1` , and with this change ` test.sh` will work with community if it is called with the argument `oss`.

Reply:
@alsastre `./test.sh oss`, not `./test.sh`. you'll have to add the `oss` argument to avoid cuke-master

Reply:
oh sorry just saw your comments above :)

Reply:
`_setup.sh` should pickup the argument though

Reply:
I checked, it does pick it up. Its only that I didn't saw where. So no fix needed only a comment on documentation to explain it